exampleconnectionservice/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+    buildToolsVersion "27.0.1"
+
+    defaultConfig {
+        applicationId "com.twilio.voice.exampleconnectionservice"
```

Comment:
package name should be `com.twilio.voice.examples.connectionservice`

Reply:
As an example see https://github.com/twilio/video-quickstart-android/tree/master/exampleVideoInvite